### ISSUE TITLE
vimrc is not installed

### ISSUE THREAD

**Issue by hupfdule** [vim-volt/volt#273]
```sh
$ # vimrc exists in $VOLTPATH, but not in the system
$ ls -l ~/volt/rc/default/vimrc
-rw-r--r-- 1 myuser myuser 9791 Mär 26 18:49 /home/myuser/volt/rc/default/vimrc
$ ls -l ~/.vimrc
ls: cannot access '/home/myuser/.vimrc': No such file or directory
$ ls -l ~/.vim/vimrc
ls: cannot access '/home/myuser/.vim/vimrc': No such file or directory
# calling volt to do a full build
$ volt build -full
[INFO] Full building /home/myuser/.vim/pack/volt/opt directory ...
[INFO] Installing vimrc and gvimrc ...
# checking for existance of vimrc (which should have been copied)
$ ls -l ~/.vimrc
ls: cannot access '/home/myuser/.vimrc': No such file or directory
$ ls -l ~/.vim/vimrc
ls: cannot access '/home/myuser/.vim/vimrc': No such file or directory
```

There is no error message and still nothing is copied.

**Comment by mrymtsk** [comment]
Can you update #272 first? 🙏 Simply closing the issue is okay if your question is answered.

The filename should be`vimrc.vim` or `gvimrc.vim`.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by hupfdule** [comment]
The problem was, that those file was named `vimrc` instead of `vimrc.vim` in my `$VOLTPATH`. It is documented in the README, but I think it should be pointed out explicitly, since it can be easily overlooked (as I did).

**Comment by tyru** [comment]
fixed in #292
